### ISSUE TITLE
A Few Questions

### ISSUE THREAD

**Issue by masterjek** [UnamSanctam/SilentCryptoMiner#1117]
Good afternoon Unam. Sorry for my English, I write through google translator. I see a new update is out and I have a couple of questions:
1. The startup code that was previously in a separate file Run PE, then was in separate libraries dll. In update 3.0.0 This code is inside the main code right?
2. I can encrypt as before xmrig.exe and already encrypted xmrig.exe compile the project into a builder using Visual Studio? Or now I need another program to compile?

**Comment by masterjek** [comment]
I managed to encrypt xmrig.exe without unpacking. If it is not hard for you then throw me here unpacked xmrig.exe. And after encryption xmrig.exe I need to squeeze it UPX?

**Comment by UnamSanctam** [comment]
You can pack it with UPX if you want though it can add some detections when it gets injected (not sure yet since the miner is new), it's not required to use UPX but it does make the file smaller. But here are both miner unpacked: [Miners.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/9505245/Miners.zip)

**Comment by masterjek** [comment]
I compiled the project into a builder with encrypted xmrig, But when I try to compile an already finished file, I get an error.
![1](https://user-images.githubusercontent.com/36513967/188855648-7bf801f0-d7ae-49bb-a6dd-b6269dc09c74.jpg)
I translated part of the text in the error - Could not find part of the path
![2](https://user-images.githubusercontent.com/36513967/188855848-e4ed9033-8495-47e9-abba-ac5a7bfbb18b.jpg)
I don't understand if this is a problem with encryption xmrig?

**Comment by parabola20** [comment]
> I compiled the project into a builder with encrypted xmrig, But when I try to compile an already finished file, I get an error. ![1](https://user-images.githubusercontent.com/36513967/188855648-7bf801f0-d7ae-49bb-a6dd-b6269dc09c74.jpg) I translated part of the text in the error - Could not find part of the path ![2](https://user-images.githubusercontent.com/36513967/188855848-e4ed9033-8495-47e9-abba-ac5a7bfbb18b.jpg) I don't understand if this is a problem with encryption xmrig?

drop the miner to disk C it helped me

**Comment by UnamSanctam** [comment]
Yeah sounds like it couldn't find the windres log file or something similar, try building it in another location, could be that your file path is too long.

**Comment by masterjek** [comment]
Thank you very much, I just put the builder in the root of drive C and the file compiled successfully. The finished file has only one virus detection. I understand the finished file is useless to encrypt? And will there always be one virus detection? Or until the file is uploaded to Virus Total?

**Comment by UnamSanctam** [comment]
>Thank you very much, I just put the builder in the root of drive C and the file compiled successfully. The finished file has only one virus detection. I understand the finished file is useless to encrypt? And will there always be one virus detection? Or until the file is uploaded to Virus Total?

It's possible to crypt the final file with a normal crypter though right now it shouldn't be necessary. It will get more detections over time but I have been uploading it to VirusTotal (because tons of people will do it anyways so it's better if I do it for testing) for a week and detections have barely increased at all so hopefully it will last for a while.

**Comment by UnamSanctam** [comment]
Hmm, should be fine, maybe something went wrong when I unpacked the xmrig.

**Comment by masterjek** [comment]
Unam Can you give me xmrig.exe Just not the one that was packed UPX And after unpacked. And the one that was not packed with UPX. I have already checked on my computer after unpacking xmrig He stops working. Even if I pack it back It still doesn't work. It seems that UPX It breaks.

**Comment by UnamSanctam** [comment]
Here is the original (that has never been packed) I believe: [xmrig.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/9507518/xmrig.zip)
